x-pack/plugin/identity-provider/src/internalClusterTest/java/org/elasticsearch/xpack/idp/saml/sp/SamlServiceProviderIndexTests.java

```diff
@@ -242,59 +240,6 @@ private void refresh() {
         future.actionGet();
     }
 
-    public static SamlServiceProviderDocument randomDocument() {
```

Comment:
moved this to utility class (in the the test sourceset) so it can be shared easier.